File: app/src/main/java/com/kickstarter/mock/factories/MessageFactory.java
package com.kickstarter.mock.factories;

import org.joda.time.DateTime;

import androidx.annotation.NonNull;

import com.kickstarter.models.Message;

public final class MessageFactory {
  private MessageFactory() {}

  public static @NonNull
  Message message() {
    return Message.builder()
      .body("")
      .createdAt(DateTime.now())
      .id(123943059L)
      .recipient(UserFactory.creator())
      .sender(UserFactory.user())
      .build();
  }
}

File: app/src/main/java/com/kickstarter/mock/factories/MessageFactory.kt
package com.kickstarter.mock.factories

import com.kickstarter.mock.factories.UserFactory.creator
import com.kickstarter.mock.factories.UserFactory.user
import com.kickstarter.models.Message
import com.kickstarter.models.Message.Companion.builder
import org.joda.time.DateTime

object MessageFactory {
    @JvmStatic
    fun message(): Message {
        return builder()
            .body("")
            .createdAt(DateTime.now())
            .id(123943059L)
            .recipient(creator())
            .sender(user())
            .build()
    }
}
